src/com/hrabrov/introduction_to_java/basic_of_software_code_development/cycle/Task03.java

```diff
@@ -18,7 +18,7 @@ public static String sumSquareHundredNumbers(long x, int h) {
         int stepToNextNumber;
         for (int i = 0; i < 100; i++) {
             if (h > 21474836) {
-                return "Increment is bigger, enter less increment";
+                return "Increment is bigger, enter increment is less";
```

Comment:
It's not clear bigger than what or less than what. Consider reworking the message to make it more clearer for the user